doc/introduction/index.rst

```diff
@@ -411,448 +391,37 @@ The Boot process proceeds as follows:
-  happening.  Currently, Virtio supports PCI/PCIe bus and MMIO bus. In
-  ACRN project, only PCI/PCIe bus is supported, and all the Virtio devices
-  share the same vendor ID 0x1AF4.
-
-**Efficient**: batching operation is encouraged
-  Batching operation and deferred notification are important to achieve
-  high-performance I/O, since notification between FE and BE driver
-  usually involves an expensive exit of the guest. Therefore, batching
-  operating and notification suppression are highly encouraged if
-  possible. This will give an efficient implementation for performance
-  critical devices.
-
-**Standard: virtqueue**
-  All the Virtio devices share a standard ring buffer and descriptor
-  mechanism, called a virtqueue, shown in Figure 6. A virtqueue
-  is a queue of scatter-gather buffers. There are three important
-  methods on virtqueues:
-
-  * ``add_buf`` is for adding a request/response buffer in a virtqueue
-  * ``get_buf`` is for getting a response/request in a virtqueue, and
-  * ``kick`` is for notifying the other side for a virtqueue to
-    consume buffers.
-
-  The virtqueues are created in guest physical memory by the FE drivers.
-  The BE drivers only need to parse the virtqueue structures to obtain
-  the requests and get the requests done. Virtqueue organization is
-  specific to the User OS. In the implementation of Virtio in Linux, the
-  virtqueue is implemented as a ring buffer structure called
-  ``vring``.
-
-  In ACRN, the virtqueue APIs can be leveraged
-  directly so users don't need to worry about the details of the
-  virtqueue. Refer to the User VM for
-  more details about the virtqueue implementations.
-
-**Extensible: feature bits**
-  A simple extensible feature negotiation mechanism exists for each virtual
-  device and its driver. Each virtual device could claim its
-  device-specific features while the corresponding driver could respond to
-  the device with the subset of features the driver understands. The
-  feature mechanism enables forward and backward compatibility for the
-  virtual device and driver.
-
-In the ACRN reference stack, we implement user-land and kernel
-space as shown in :numref:`virtio-framework-userland`:
-
-.. figure:: images/virtio-framework-userland.png
-   :width: 600px
-   :align: center
-   :name: virtio-framework-userland
-
-   Virtio Framework - User Land
-
-In the Virtio user-land framework, the implementation is compatible with
-Virtio Spec 0.9/1.0. The VBS-U is statically linked with the Device Model,
-and communicates with the Device Model through the PCIe interface: PIO/MMIO
-or MSI/MSI-X. VBS-U accesses Virtio APIs through the user space ``vring`` service
-API helpers. User space ``vring`` service API helpers access shared ring
-through a remote memory map (mmap). HSM maps User VM memory with the help of
-ACRN Hypervisor.
-
-.. figure:: images/virtio-framework-kernel.png
-   :width: 600px
-   :align: center
-   :name: virtio-framework-kernel
+* :ref:`introduction`
```

Comment:
Remove this line, the file is pointing to itself.

Reply:
doh